.github/workflows/codespell.yml

```diff
@@ -0,0 +1,10 @@
+# https://pypi.org/project/codespell
+name: codespell
+on: [pull_request, push]
+jobs:
+  codespell:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - run: pip install codespell
+      - run: codespell --ignore-words-list=braket,ket
```

Comment:
Can you add `multline` and `merrors` to this list to avoid the false positives?

Reply:
```suggestion
      - run: codespell --ignore-words-list=braket,ket,merror,multline
```